app/build.gradle

```diff
@@ -45,6 +45,23 @@ android {
         }
     }
 
+    flavorDimensions "default"
+
+    productFlavors {
+        fdroid {
+            dimension "default"
+            applicationIdSuffix ".fdroid"
+            versionNameSuffix "-FDroid"
+            ext.enableCrashlytics = false
```

Comment:
This is not how it works, you need to change the dependency and also remove all references to crashlytics in this product flsvor

Reply:
@iamareebjamal crashlytics is not acceptable in fdroid so instead on adding 
 playStoreImplementation("com.crashlytics.sdk.android:crashlytics:${rootConfiguration.crashlyticsVersion}") {
        transitive = true
    }
wouldn't it work?

Reply:
This change is not visible in PR

Reply:
Compile fdroid apk and upload it here

Reply:
There are no code changes in the PR. Is the crashlytics code used nowhere?

Also, Travis build script needs to be changed so that both apps are generated

Reply:
@iamareebjamal the crashlytics code is only used in MainApplication.java file only and it is commented out

Reply:
Then we are not using crashlytics and it can be removed

Reply:
@iamareebjamal should I close this PR then ?